core/debugger/remote_debugger.cpp

```diff
@@ -78,18 +78,22 @@ struct RemoteDebugger::NetworkProfiler {
 
 	int bandwidth_usage(const Vector<BandwidthFrame> &p_buffer, int p_pointer) {
 		int total_bandwidth = 0;
-
-		uint32_t timestamp = OS::get_singleton()->get_ticks_msec();
-		uint32_t final_timestamp = timestamp - 1000;
-
-		int i = (p_pointer + p_buffer.size() - 1) % p_buffer.size();
-
-		while (i != p_pointer && p_buffer[i].packet_size > 0) {
-			if (p_buffer[i].timestamp < final_timestamp) {
-				return total_bandwidth;
+		
+		// No frames no usage.
+		if(p_buffer.size()) // Not zero
+		{
```

Comment:
little style issue here
```suggestion
		if (p_buffer.size()) {
```

Reply:
Where do I find the style guide/standard?

Reply:
https://github.com/godotengine/godot/tree/master/misc/hooks
this will help.

Reply:
See [Using clang-format locally](https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#using-clang-format-locally) as well.